king_phisher/serializers.py

```diff
@@ -283,12 +284,15 @@ def to_elementtree_subelement(parent, tag, value, attrib=None):
 		if case(str):
 			type_ = 'string'
 			break
+		if case(unicode):
```

Comment:
`unicode` isn't defined in Python 3, this needs to check the version as appropriate and set a `str_type` or something.